app/lib/components/tab.js

```diff
@@ -41,7 +49,7 @@ export default class Tab extends Component {
             isLast && 'textLast',
             isActive && 'textActive'
           ) }
-          onClick={ this.props.isActive ? null : this.props.onSelect }>
+          onClick={ this.handleClick.bind(this) }>
```

Comment:
Binding every render has a performance penalty. Maybe create a bound version of `handleClick` in the constructor?